packages/gestalt/src/Box.js

```diff
@@ -32,12 +32,14 @@ import {
   toProps,
 } from './style.js';
 import {
+  // flowlint untyped-import:off
```

Comment:
Turning the rule off for the imports in this file as I need to research @chrislloyd 's "functors" from `transforms.js` more before typing them properly